pages/privacy.js

```diff
@@ -0,0 +1,254 @@
+          <ListItem>To allow you to participate in interactive features of our Service when you choose to do so</ListItem>
+          <ListItem>To provide customer support</ListItem>
+          <ListItem>To gather analysis or valuable information so that we can improve our Service</ListItem>
+          <ListItem>To monitor the usage of our Service</ListItem>
+          <ListItem>To detect, prevent and address technical issues</ListItem>
+        </UnorderedList>
+        <Divider />
+        <FAQHeading>Retention of Data</FAQHeading>
+        <PrivacyTextBlock>
+        We will retain your Personal Data only for as long as is necessary for the purposes set out in this Privacy Policy. We will retain and use your Personal Data to the extent necessary to comply with our legal obligations (for example, if we are required to retain your data to comply with applicable laws), resolve disputes, and enforce our legal agreements and policies.
+        </PrivacyTextBlock>
+        <PrivacyTextBlock>
+        We will also retain Usage Data for internal analysis purposes. Usage Data is generally retained for a shorter period of time, except when this data is used to strengthen the security or to improve the functionality of our Service, or we are legally obligated to retain this data for longer time periods.
+        </PrivacyTextBlock>
+        <Divider />
+        <FAQHeading>Transfer Of Data</FAQHeading>
+        <PrivacyTextBlock>
+        Your information, including Personal Data, may be transferred to — and maintained on — computers located outside of your state, province, country or other governmental jurisdiction where the data protection laws may differ than those from your jurisdiction.
+        </PrivacyTextBlock>
+        <PrivacyTextBlock>
+        We will take all steps reasonably necessary to ensure that your data is treated securely and in accordance with this Privacy Policy and no transfer of your Personal Data will take place to an organization or a country unless there are adequate controls in place including the security of your data and other personal information.
+        </PrivacyTextBlock>
+        <PrivacyTextBlock>
+        Your consent to this Privacy Policy followed by your submission of such information represents your agreement to that transfer.
+        </PrivacyTextBlock>
+        <Divider />
+        <FAQHeading>Disclosure Of Data</FAQHeading>
+        <UnorderedList>
+          <ListItem>
+          Business Transaction. If we are involved in a merger, acquisition or asset sale, your Personal Data may be transferred. We will provide notice before your Personal Data is transferred and becomes subject to a different Privacy Policy.
+          </ListItem>
+          <ListItem>
+          Disclosure for Law Enforcement. Under certain circumstances, we may be required to disclose your Personal Data if required to do so by law or in response to valid requests by public authorities (e.g. a court or a government agency).
```

Comment:
would love this to not be the case. are you set on this

Reply:
this is required by law, unsure if we can avoid this. if it's required by law, it needs to be in the privacy policy to let users know

Reply:
changed to just "if required by law"